taccsite_cms/_settings/email.py

```diff
@@ -0,0 +1,18 @@
+"""Configure email settings (for apps e.g. djangocms_forms)"""
+
+########################
+# DJANGO
+# https://docs.djangoproject.com/en/2.2/topics/email/
+########################
+
+# Development, Staging, Production
+# https://confluence.tacc.utexas.edu/x/coR9E
+# EMAIL_HOST = "relay.tacc.utexas.edu"
+# DEFAULT_FROM_EMAIL = 'no-reply@...' # where ... is project's production domain
```

Comment:
I added these to APCD settings in https://github.com/TACC/Core-CMS-Custom/pull/4.